rust/src/lightning.rs

```diff
@@ -331,6 +333,11 @@ pub fn setup(
     user_config
         .channel_handshake_limits
         .force_announced_channel_preference = false;
+    // By setting this to 0 we indicate we accept 0-conf channels.
+    user_config.channel_handshake_config.minimum_depth = 0;
+    // By setting `manually_accept_inbound_channels` to `true` we need to manually confirm every
+    // inbound channel request.
+    user_config.manually_accept_inbound_channels = true;
```

Comment:
❓ Is this actually necessary?

Below you're just accepting unconditionally anyway.

Reply:
Yes, otherwise the event `OpenChannelRequest` won't be fired